code/modules/mob/living/carbon/human/human.dm

```diff
@@ -873,6 +873,126 @@
 
 			log_combat(src, T, "curbstomped")
 			src.is_busy = FALSE
+/mob/living/carbon/human/MouseDrop(mob/over)//Groinkick added based off of above curbstomp code

```

Comment:
That's not how this works. Add your new code to the existing proc.

Honestly this will require a bit more of a rewrite.

Reply:
instead of creating a new one each time just use the corresponding code, correct

Reply:
Still contains a lot of unnecessary copypasta.

Reply:
ok, ill have a look about in the code